storage/cleanup.go

```diff
@@ -89,7 +89,13 @@ func (m *cleanupManager) Cleanup(t time.Time) error {
 			"encountered errors when cleaning up fileset files for %v: %v", t, err))
 	}
 
-	commitLogStart, commitLogTimes := m.commitLogTimes(t)
+	commitLogStart, commitLogTimes, err := m.commitLogTimes(t)
```

Comment:
Could you verify we have unit test coverage to verify that `Cleanup()` behaves as expected in both cases ~ when GetOwnedNamespaces returns error, and when it doesn't.

Reply:
Looks like happy path was already tested so I added a unit test for the case where commitLogTimes returns an error